### PR TITLE
Z_PROBE_ENDSTOP

### PR DESCRIPTION
**Pull request by croadfeldt**
Cleaned up PR for Z_PROBE_ENDSTOP.

### CODE REVIEW

File: Marlin/language.h [hunk 1]
```diff
@@ -138,6 +138,7 @@
 #define MSG_Z_MIN                           "z_min: "
 #define MSG_Z_MAX                           "z_max: "
 #define MSG_Z2_MAX                          "z2_max: "
+#define MSG_Z_PROBE							"z_probe: "
```
Comment (thinkyhead) [inline Marlin/language.h]: Oops, spaces here, not tabs.

**Comment by thinkyhead** [comment]
Overall, this is great. We're jumping into a bug-fix mode for a while, so this is on hold till the code is more solid. Later, we might think about calling this `Z_PROBE_SWITCH`, because it's not _really_ an end-stop.

**Comment by croadfeldt** [comment]
Rock on, Z_PROBE_SWITCH works for me. I see my tabs returned in the language file. :) Glad to help out. Looking at the offset thing still. I think what I want to do is already in the code, just need to correctly configure it and then update the documentation a bit.

**Comment by Natealus** [comment]
@thinkyhead Wouldn't this help for all the fixes you're doing to bed leveling? I've wondered for a few days if the fact that Z_Probe and Z_MIN_ENDSTOP were kind of the same object in the code, that if it might be causing issues?

**Comment by thinkyhead** [comment]
@Natealus Well, this will certainly help eliminate that as a contributing factor. It doesn't amount to a drastic alteration, so I will merge this as part of the effort to deal with any probe=endstop issues, namely that when the probe is an endstop the Z axis is reset to the home position when homed.

**Pull request merged**